velox/expression/signature_parser/tests/ParseTypeSignatureTest.cpp

```diff
@@ -54,19 +79,25 @@ TEST(ParseTypeSignatureTest, array) {
 }
 
 TEST(ParseTypeSignatureTest, map) {
-  auto signature = parseTypeSignature("map(bigint, double)");
+  auto signature = *parseTypeSignature("map(interval day to second, double)");
   ASSERT_EQ(signature.baseName(), "map");
   ASSERT_EQ(signature.parameters().size(), 2);
 
   auto key = signature.parameters()[0];
-  ASSERT_EQ(key.baseName(), "bigint");
+  ASSERT_EQ(key.baseName(), "interval day to second");
   ASSERT_EQ(key.parameters().size(), 0);
 
   auto value = signature.parameters()[1];
   ASSERT_EQ(value.baseName(), "double");
   ASSERT_EQ(value.parameters().size(), 0);
 }
 
+TEST(ParseTypeSignatureTest, row) {
+  auto signature = *parseTypeSignature("row(v)");
+  ASSERT_EQ(signature.baseName(), "row");
+  ASSERT_EQ(signature.parameters().size(), 1);
+}
+
```

Comment:
the missing feature from the existing code was to support quoted named row fields. Could you add a few tests to ensure that works now?